docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,373 @@
+---
+title: Telemetry (alpha)
+---
+
+## Kyma's telemetry component
+
+The focus of Kyma's observability functionality is the pre-integrated collection of telemetry data from the users' workloads, and making that data available for further analysis and storage in backends of any kind and location. The telemetry component provides the configurable collection and shipment components, and with that, separates them explicitly from the storage and analysis in a specific backend system. Lightweight in-cluster backends can be installed with dedicated observability components.
+
+For now, the telemetry component supports the logging domain only. You get a log collector (Fluent Bit) and you can configure the log shipment with external systems using runtime configuration with a dedicated Kubernetes API (CRD). With that, for example, you can integrate with vendors like [VMWare](https://medium.com/@shrishs/log-forwarding-from-fluent-bit-to-vrealizeloginsightcloud-9eeb14b40276) using generic outputs, or with any vendor via a [fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa) using the forward output. Kyma's optional `logging` component complements the telemetry component, providing `Loki` as pre-configured log backend.
+
+The telemetry component is in an alpha state and needs an [explicit installation](./../../../04-operation-guides/operations/obsv-00-enable-configurable-logging.md).
+
+## Architecture
+
+### Telemetry component
+
+The telemetry component ships [Fluent Bit](https://fluentbit.io/) as a log collector. It is responsible for collecting all application logs of the cluster workload reliably and shipping them to a backend.
+
+The only prerequisite in your application is to log to `stdout` or `stderr`.
+
+![Telemetry Architecture](./assets/telemetry-logging-arch.drawio.svg)
+
+1. Container logs are stored by Kubernetes container runtime under the `var/log` directory and its subdirectories.
+2. The collector runs as a Daemon Set (one instance per node), detects any new log files in the folder, and tails them using a filesystem buffer for reliability.
```

Comment:
"the collector" means "Fluent Bit", right? Because we name it explicitly in line 26. We should choose one consistent term.